lib/src/entrypoint.dart

```diff
@@ -246,6 +256,11 @@ class Entrypoint {
     required PubAnalytics? analytics,
     bool onlyReportSuccessOrFailure = false,
   }) async {
+    if (!onlyReportSuccessOrFailure && hasPubspecOverrides) {
+      log.warning(
+          'Warning: pubspec.yaml has overrides from pubspec_overrides.yaml');
```

Comment:
Perhaps use `${pubspec_overrides_path}` such that the relative path comes out right.

Reply:
I'm not sure that's useful since `pubspec_overrides.yaml` is always going to be right next to `pubspec.yaml`.

Reply:
Yes, but `pubspec.yaml` is not always in the current working directory (if pub is invoked with `--directory`)

Reply:
Done